service.datadog.yaml

```diff
@@ -0,0 +1,56 @@
+schema-version: v2.2
+dd-service: datadog-operator
+team: container-ecosystems
+contacts:
+  - type: slack
+    contact: https://dd.slack.com/archives/C037CDX0WJV
+  - type: email
+    contact: team-containerecosystems@datadoghq.com
+links:
+  - name: Runbook
+    type: runbook
+    url: https://datadoghq.atlassian.net/wiki/spaces/ContEco/pages/2446754987/Runbooks
+  - name: datadog-operator
+    type: repo
+    provider: github
+    url: https://github.com/DataDog/datadog-operator
+  - name: deployment
+    type: repo
+    provider: github
+    url: https://github.com/DataDog/k8s-datadog-agent-ops
+  - name: internal-chart
+    type: repo
+    provider: github
+    url: https://github.com/DataDog/k8s-resources/tree/master/k8s/datadog-operator
+  - name: internal-image
+    type: repo
+    provider: github
+    url: https://github.com/DataDog/images/tree/master/datadog-operator
+  - name: On Call documentation
+    type: doc
+    provider: confluence
+    url: https://datadoghq.atlassian.net/wiki/spaces/ContEco/pages/2445645960/On+Call
+tags:
+  - app:datadog-operator
+  - service:datadog-operator
+  - team:container-ecosystems
+integrations:
+  pagerduty:
+    service-url: https://datadog.pagerduty.com/service-directory/PFZC0QZ
+extensions:
+  datadoghq.com/sdp:
+    conductor:
```

Comment:
this currently does nothing, first i want to verify that the service is ack'd by conductor